db/migrate/20220314144443_add_color_fields_to_site_customization_pages.rb

```diff
@@ -0,0 +1,6 @@
+class AddColorFieldsToSiteCustomizationPages < ActiveRecord::Migration[5.2]
+  def change
+    add_column :site_customization_pages, :background_color, :text
+    add_column :site_customization_pages, :font_color, :text
```

Comment:
I know banners are `text` as well :relieved:, but was wondering whether it makes sense :thinking:. A `string` field would probably be more appropriate considering the text is going to be just 7 characters long. What do you think? :thinking: